image/templates/common/setup-scanner-v4.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Sets up the Central deployment to use Scanner V4
+
+# shellcheck disable=SC1083
+KUBE_COMMAND=${KUBE_COMMAND:-{{.K8sConfig.Command}}}
+NAMESPACE="${ROX_NAMESPACE:-stackrox}"
+
+${KUBE_COMMAND} -n "$NAMESPACE" patch deploy/central --patch-file <(cat <<EOF
+spec:
+  template:
+    spec:
+      containers:
+      - name: central
+        env:
+        - name: "ROX_SCANNER_V4"
+          value: "true"
+EOF
```

Comment:
FYI, `kubectl` and `oc` have a cmd to update env vars directly, could simplify this a bit:

```suggestion
${KUBE_COMMAND} -n "$NAMESPACE" set env deploy/central ROX_SCANNER_V4=true
```

Reply:
I didn't know that, thanks! It seems though that it doesn't have the option to only change the central container of the deployment (not that it would matter that much if we add the env var to all containers).

Reply:
It looks to have the option (but agreed no harm in setting it on all in this case):
```
    -c, --containers='*':
	The names of containers in the selected pod templates to change - may use wildcards
```